src/Microsoft.TestPlatform.CoreUtilities/Helpers/Interfaces/IDotnetHostHelper.cs

```diff
@@ -6,7 +6,7 @@ namespace Microsoft.VisualStudio.TestPlatform.CrossPlatEngine.Helpers.Interfaces
     /// <summary>
     /// Helper class for getting info about dotnet host.
     /// </summary>
-    internal interface IDotnetHostHelper
+    public interface IDotnetHostHelper
```

Comment:
Why should this be public? I would suggest moving this into TestHostProvider implementation.

We need to discuss how DataCollector can consume this.